프로그래머스/1/12982. 예산/예산.js

```diff
@@ -0,0 +1,13 @@
+function solution(d, budget) {
+    const amountArr = d.slice().sort((a, b) => a - b);
```

Comment:
```suggestion
    const sortedRequests = requests.sort((a, b) => a - b);
```
sort() 함수는 원본 배열을 변경하지만, 이 문제의 경우 원본 배열을 유지할 필요가 없으므로 바로 정렬해도 될것 같아요 !